Dockerfile.arm

```diff
@@ -0,0 +1,46 @@
+## package #####################################################################
```

Comment:
If we have the build args now, why is the separate Dockerfile necessary?

Reply:
I did some tests and using the current `Dockerfile` for building arm takes a fair amount of time specially because we use `make build` in our development and package targets(for arm64 it can take 20 minutes sometimes even more for each of the make build commands finish) so I added this one to make things faster.
What do you think?

Reply:
That's fair enough, I guess let's keep an open eye to make sure they don't drift (unlikely) or if we can make improvements using new container build processes